.github/workflows/ci.yaml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+    push:
+    pull_request:
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        name: PHPUnit Tests
+        strategy:
+            fail-fast: true
```

Comment:
should be removed IMO. If tests fail on one of the jobs of the matrix, we still want to run the others to know whether the failure is only for that combination or no.

Reply:
The default value is [true](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast), I've set it to false now.